### PR TITLE
Update data-lake-overview.md with line separator

### PR DESCRIPTION
**Pull request by tatisled**
Added a line separator to distinguish the section title from the medallion layers list before.

### CODE REVIEW

**Comment by ttorble** [comment]
@mboswell 

Can you review the proposed changes?

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/scenarios/cloud-scale-analytics/best-practices/data-lake-overview.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ The Azure Data Lake is a massively scalable and secure data storage for high-per
 Each storage account within your data landing zone stores data in one of three stages, which align to a [medallion architecture](/azure/databricks/lakehouse/medallion):
 
 - Raw data (bronze)
-- Enriched (silver) and curated data (gold)
-- Development data lakes
+- Enriched data (silver)
+- Curated data (gold)
+
+## Development data lakes
```
Comment (mboswell) [inline docs/scenarios/cloud-scale-analytics/best-practices/data-lake-overview.md:22]: This shouldn't be a headline. It should be a line like it was in the previous version

**Review by mboswell — CHANGES_REQUESTED** [review]
Please update per feedback

File: docs/scenarios/cloud-scale-analytics/best-practices/data-lake-overview.md (hunk 1 above)
Comment (tatisled) [reply docs/scenarios/cloud-scale-analytics/best-practices/data-lake-overview.md:22]: Hi @mboswell 
Thank you for reviewing my PR.

In the previous version "Development data lakes" was a part of data stages of medallion architecture.
Are you saying that this is the correct version?
```
Each storage account within your data landing zone stores data in one of three stages, which align to a medallion architecture:

- Raw data (bronze)
- Enriched (silver) and curated data (gold)
- Development data lakes
```

**Comment by mboswell** [comment]
I am saying the text is correct as it was. This is because it relates to the architecture in https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/cloud-scale-analytics/architectures/data-landing-zone#data-lake-services and should not be changed.

File: docs/scenarios/cloud-scale-analytics/best-practices/data-lake-overview.md (hunk 1 above)
Comment (ckittel) [inline docs/scenarios/cloud-scale-analytics/best-practices/data-lake-overview.md:22]:
```suggestion
- Development data lakes
```

**Pull request closed without merging**